code/modules/mob/dead/observer/observer_say.dm

```diff
@@ -5,8 +5,12 @@
 /mob/dead/observer/get_message_mods(message, list/mods)
 	var/key = message[1]
 	if((key in GLOB.department_radio_prefixes) && length(message) > length(key) + 1 && !mods[RADIO_EXTENSION])
-		mods[RADIO_KEY] = LOWER_TEXT(message[1 + length(key)])
-		mods[RADIO_EXTENSION] = GLOB.department_radio_keys[mods[RADIO_KEY]]
+		mods[RADIO_KEY] = message[1 + length(key)]
+		// intelligently use lowercase if there doesn't exist an uppercase version of that letter in department_radio_keys
```

Comment:
I don't think we should have uppercase/lowercase keys people already don't remember these